lib/rake/circle.rake

```diff
@@ -4,6 +4,10 @@ require 'cdo/git_utils'
 RUN_UI_TESTS_TAG = '[test ui]'
 RUN_ALL_TESTS_TAG = '[test all]'
 
+def random_string_of_length(n)
+  rand(36**n).to_s(36)
+end
```

Comment:
Would it make more sense to use an existing random string generator? Maybe something like SecureRandom.uuid? (not sure what's standard in ruby land - that's just something I found). I realize we don't really care about it being truly random, so maybe what you have is fine.